packages/ui-components/src/index.js

```diff
@@ -111,4 +111,4 @@ export { TooltipContent } from "./components/TooltipContent/index.js"
 export { TooltipTrigger } from "./components/TooltipTrigger/index.js"
 export { TopNavigation } from "./components/TopNavigation/index.js"
 export { TopNavigationItem } from "./components/TopNavigationItem/index.js"
-export * from "./components/Form/index.js"
+export * from "./components/Form/index"
```

Comment:
@andypf Any reason we've two exports for `Form`? I see this line was introduced to improved bundle size? But just wondering if we need two? 

Also, is it correct that FormHint isn't exported here?

Reply:
No :)

Reply:
good catch 👍